doctrine.rst

```diff
@@ -568,6 +569,12 @@ a controller, this is pretty easy. Add the following method to the
         return new Response('Saved new product with id '.$product->getId());
     }
 
+    // you can also receive the $em as an argument
+    public function editAction(EntityManagerInterface $em)
```

Comment:
I'll update the whole Doctrine section in another PR with more of this fancy type-based stuff :)